extensions/oidc/deployment/src/main/java/io/quarkus/oidc/deployment/devservices/keycloak/DevServicesConfig.java

```diff
@@ -87,6 +87,9 @@ public class DevServicesConfig {
      * A comma-separated list of class or file system paths to Keycloak realm files.
      * This list is used to initialize Keycloak.
      * The first value in this list is used to initialize default tenant connection properties.
+     * To learn more about creating Keycloak realm files consult the <a href="https://www.keycloak.org/server/importExport">documentation</a>
+     * <p>
+     * An example of such a file system path is {@code src/resources/realm.json}
```

Comment:
Thanks @bulldog98, but I'm not sure an example like `src/resources/realm.json` is useful, file system resource is a self describing thing. The next thing we'll get QE asking what is this path relative to, did we mean `src/main/resources`, and if yes, why just don't say `realm.json` since it will be on the classpath. IMHO this line should be dropped. IMHO Only a link to the Keycloak docs should remain

Reply:
Thanks for the input @sberyozkin, I applied this suggestion